content/blog/2019-02-17-Incubator.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Software Incubator Workshop: A Synthesis"
+date: 2019-02-17T17:44:32-06:00
+author: "Sebastian P. Benthall"
+---
+
+*Thanks for comments from: Dan Katz, Suresh Marru, Abby Cabunoc Mayes, Micaela S. Parker, Danielle Robinson, and Nic Weber.*
+
+Research software is software created by scholarly researchers. Often researchers produce code that is ‘open’, in alignment with scientific norms. But this code and the research it supports is not always “sustainable”, because it does not have a viable community of developers and users working with it in an ongoing way. For open source software in general, there are several detailed, existing taxonomies of successful open source community models, including the[ Apache Project Maturity Model](http://community.apache.org/apache-way/apache-project-maturity-model.html) and Mozilla’s [Open Source Archetypes](https://blog.mozilla.org/wp-content/uploads/2018/05/MZOTS_OS_Archetypes_report_ext_scr.pdf). Are there specific patterns of open community challenges and growth for research software that an incubator could address?
```

Comment:
The post starts with software but immediately refers to code in the next sentence. It would be good to keep consistent.